### DISCUSSION TITLE
Session issue?

### DISCUSSION THREAD

**Discussion by lyandr** [requarks/wiki#3482] [Help / Questions]
**Question**
After sometimes without connecting to wikijs, i have the following errors:
`Oops, something went wrong...
select "sess" from "sessions" where "sid" = $1 and CAST($2 as timestamp with time zone) <= expired - could not seek to end of file "base/16385/17038": Stale file handle
`
If i open a new private window on the browser it is ok and i can login.

This issue seems to be related to sessions? Did i miss something on wikijs configuration?

**Host Info:**
OS: Linux - Alpine Linux (linux) 3.11.6 x64
Wiki.js version: 2.5.144
Database engine: 9.6.10

**Comment by NGPixel** [comment]
Looks like a configuration issue with your database
